docs/mlir/getting_started.md

```
@@ -0,0 +1,321 @@
+
+## The Setup
+
+To work with MLIR in MQT Core (run the pipeline, run tests, or inspect IR), you need to build from source.
+Follow the {doc}`installation guide <../installation>`, specifically the "Building from Source for Performance" and "Development Setup" sections.
+The default CMake configuration builds the MLIR pipeline and the compiler unit tests.
+Run `cmake --build build` to get binaries and tests.
+
+To run the compiler test suite from the project root, use CTest (with a regex to filter for the compiler tests):
+
+```console
+$ ctest --test-dir build -R "mqt-core-mlir-unittests-compiler"
```

markdownlint-cli2 (0.21.0)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)